utils.rb

```diff
@@ -0,0 +1,55 @@
+def add_it(initial)
+  sum = initial
+  my_each { |e| sum += e }
+  sum
+end
```

Comment:
This method seems to be adding the first value of the array two times when no initial value is given in the `inject` method.

```ruby
irb(main):002:0> [1,2,3,4,5,56,7,8,9,234].inject(:+)
#=> 329
irb(main):003:0> [1,2,3,4,5,56,7,8,9,234].my_inject(:+)
#=> 330

irb(main):005:0> [19,2,3,4,5,56,7,8,9,234].inject(:+)
#=> 347
irb(main):006:0> [19,2,3,4,5,56,7,8,9,234].my_inject(:+)
#=> 366
```